src/components/UserDetailSections/UserInfo/UserInfo.test.js

```diff
@@ -2,19 +2,34 @@ import { screen } from '@folio/jest-config-stripes/testing-library/react';
 import '__mock__/stripesComponents.mock';
 
 import renderWithRouter from 'helpers/renderWithRouter';
+import { useStripes } from '@folio/stripes/core';
 import UserInfo from './UserInfo';
+import { useProfilePicture } from './hooks';
+
+import profilePicData from '../../../../test/jest/fixtures/profilePicture';
+
+jest.unmock('@folio/stripes/components');
+jest.unmock('@folio/stripes/util');
+
+jest.mock('@folio/stripes/core', () => ({
+  ...jest.requireActual('@folio/stripes/core'),
+  useStripes: jest.fn(() => ({
+    hasPerm: () => true,
+  })),
+}));
```

Comment:
The `useStripes` is already mocked in `stripesCore.mock.js`.

Reply:
cleanedup